spec/features/category_spec.rb

```diff
@@ -36,4 +41,10 @@
     find_link("Destroy").click
     expect(page).to have_content "Category was successfully destroyed."
   end
+
+  scenario 'should show all carriers' do 
```

Comment:
Would it be a good idea to test that carrier is a link and that user can navigate to the "carrier#show" page